gensrc/proto/olap_file.proto

```diff
@@ -181,6 +181,13 @@ enum KeysType {
 message DeletePredicatePB {
     required int32 version = 1;
     repeated string sub_predicates = 2;
+    repeated InPredicatePB in_predicates = 3;
+}
+
+message InPredicatePB {
```

Comment:
For  protobuf, we prefer optional than required, because we maybe remove or rename `column_name` from `InPredicatePB`. Please change all `required ` to `optional`.

Reply:
done